Core/include/Acts/Seeding/SeedFinderConfig.hpp

```diff
@@ -57,13 +57,23 @@ struct SeedFinderConfig {
   float rMinMiddle = 60.f * Acts::UnitConstants::mm;
   float rMaxMiddle = 120.f * Acts::UnitConstants::mm;
 
+  // z of last layers to avoid iterations
+  std::pair<float, float> zOutermostLayers{-2700 * Acts::UnitConstants::mm,
+                                           2700 * Acts::UnitConstants::mm};
+
   // cut to the maximum value of delta z between SPs
   float deltaZMax =
       std::numeric_limits<float>::infinity() * Acts::UnitConstants::mm;
 
   // enable cut on the compatibility between interaction point and SPs
   bool interactionPointCut = false;
 
+  // cut on bottom SPs in a certain (r, eta) region of the detector for fast
+  // seeding
+  bool fastTrackingCut = false;
+  float fastTrackingRMin = 50. * Acts::UnitConstants::mm;
```

Comment:
being a bottom SP requirement and being a general acceptance on the radius, would it make sense not to fill the grid with sps with a radius < 50 mm ? Or is this RMin requirement something else?

Reply:
my point here is that if we have less space point in the grid, we iterate on less space points and we have no need to have an extra if statement. Much faster.

Actually, I would take a look at all the cuts we apply in the seeding process, to see if something can be moved way ahead and avoid pointless iterations on space point that will never satisfy any cut

Reply:
this is a cut on the bottom-middle combinations in which the bottom has a radius smaller than fastTrackingRMin and cotTheta = deltaZ * iDeltaR in a certain range. So I think we cannot move these cuts to the grid filling

Reply:
all of the cuts inside getCompatibleDoubltes depend on the combination of two SPs